lib/utils/sdJwt.ts

```diff
@@ -1,6 +1,7 @@
-import { decodeDisclosuresInPayload } from '@sd-jwt/decode';
-import { getDisclosuresForPresentationFrame, PresentationFrame } from '@sd-jwt/present';
-import { Base64url } from '@sd-jwt/utils';
+import { getSDAlgAndPayload, unpackObj } from '@sd-jwt/decode';
+import { createHashMappingForSerializedDisclosure, selectDisclosures } from '@sd-jwt/present';
+import { PresentationFrame } from '@sd-jwt/types';
+import { Uint8ArrayToBase64Url } from '@sd-jwt/utils';
```

Comment:
Is this a function? Why does it start with a capital?

Reply:
I probably made a mistake while writing down from type. It doesn't mean much. Should I use `as` and change it to lowercase?

```ts
import { Uint8ArrayToBase64url as uint8ArrayToBase64url } from '@sd-jwt/utils'
```